packages/ast-spec/src/expression/ArrayExpression/spec.ts

```diff
@@ -1,8 +1,9 @@
 import type { AST_NODE_TYPES } from '../../ast-node-types';
 import type { BaseNode } from '../../base/BaseNode';
+import type { SpreadElement } from '../../element/SpreadElement/spec';
 import type { Expression } from '../../unions/Expression';
 
 export interface ArrayExpression extends BaseNode {
   type: AST_NODE_TYPES.ArrayExpression;
-  elements: Expression[];
+  elements: (Expression | SpreadElement)[];
```

Comment:
I'll defer to @bradzacher here because of the recent work with snapshots. But this LGTM to in general!

Reply:
This is correct.
We won't require any one add snapshots for changes until the migration is finished.
There are still a few bits that might need some ironing out